library/java/io/envoyproxy/envoymobile/engine/JniLibrary.java

```diff
@@ -315,6 +315,11 @@ protected static native int recordHistogramValue(long engine, String elements, b
    */
   public static native String brotliConfigInsert();
 
+  /**
+   * Provides a configuration insert that may be used to enable socket tagging.
+   */
+  public static native String socketTagConfigInsert();
```

Comment:
so you know I'd envisioned this being done as brotli was _before_ my brotli PR, and having cronet just add this as a custom filter.  I'm 100% with doing the extras but it comes with a higher API/docs burden
though really it's worth doccing anyway.  :-)

Reply:
OH! Shoot! I misunderstood. Hm. I'm happy to un-do part of this if that would be better?

Reply:
nah, fine to leave in unless Lyft folks object.